libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -561,6 +562,14 @@ bool AP_Arming::system_checks(bool report)
             check_failed(ARMING_CHECK_SYSTEM, report, "Param storage failed");
             return false;
         }
+
+        for (uint8_t i=0; i<RANGEFINDER_MAX_INSTANCES; i++) {
+            AP_RangeFinder_Backend *sensor = RangeFinder::get_singleton()->get_backend(i);
+            if (sensor != nullptr && (sensor->status() == RangeFinder::RangeFinder_NotConnected || sensor->status() == RangeFinder::RangeFinder_NoData)) {
+                check_failed(ARMING_CHECK_SYSTEM, report,"Rangefinder %d failed", i);
```

Comment:
Add 1 to `i` so that the printed number matches the numbers used when setting up rangefinders.